examples/k8s/main.go

```diff
@@ -49,27 +55,28 @@ func main() {
 	confFile, err := os.ReadFile(configDir + confFileName)
 	if err != nil {
 		log.Fatal("%s", err)
-		os.Exit(1)
+		return err
 	}
 
 	// Network and node configs
 	var allConfig allConfig
 	if err := json.Unmarshal(confFile, &allConfig); err != nil {
 		log.Fatal("%s", err)
-		os.Exit(1)
+		return err
 	}
 
 	// TODO maybe do networkConfig validation
 	networkConfig, err := readConfig(allConfig)
 	if err != nil {
 		log.Fatal("error reading configs: %s", err)
-		os.Exit(1)
+		return err
 	}
 	networkConfig.ImplSpecificConfig = allConfig.K8sConfig
 
 	level, err := logging.ToLevel(networkConfig.LogLevel)
 	if err != nil {
-		log.Warn("Invalid log level configured: %s", err)
+		log.Fatal("couldn't parse log: %s", err)
+		return err
```

Comment:
I was not exiting at this point - I understood this to be such a low-impact error that I decided to simply ignore this.

Reply:
If `err != nil` than `level` isn't well-defined, and this should never happen, so we should exit here